sdk/src/opentelemetry/trace/tracer_provider.cc

```diff
@@ -0,0 +1,18 @@
+#include "opentelemetry/sdk/trace/tracer_provider.h"
+
+namespace opentelemetry
+{
+namespace sdk
+{
+namespace trace
+{
+Tracer *const TracerProvider::GetTracer(nostd::string_view library_name,
+                                        nostd::string_view library_version,
+                                        const opentelemetry::trace::TracerOptions &tracer_options)
+{
+  tracers_.emplace_back(new Tracer(library_name, library_version, tracer_options));
+  return tracers_.back().get();
```

Comment:
Why are we using a vector and constructing a new tracer for each call to GetTracer?

Reply:
This was originally a map based on name + version pair, as that deduping makes more sense to me. I changed it to a vector due to [this conversation](https://github.com/open-telemetry/opentelemetry-cpp/pull/24#discussion_r363914305). I'll convert this back into a map, since it seems to be the consensus.